tools/autograd/gen_python_functions.py

```diff
@@ -25,8 +25,8 @@
     'index',
     '_indexCopy_', 'max_values', 'min_values', 'argmax', 'argmin',
     '_cumsum.*', '_cumprod.*', '_sum.*', '_prod.*', '_th_.*',
-    'arange.*', 'range.*', '_gesv.*', '_getri.*', '_inverse.*', 'slice',
-    'randint(_out)?',
+    'arange.*', 'range.*', '_gesv.*', '_getri.*', '_inverse.*', '_potrs.*',
```

Comment:
For the future: maybe we should start naming these _linalg_gesv*, _linalg_getri_*, _linalg_inverse* etc so we don't have to keep on updating this file.